packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -425,34 +425,32 @@ const ReleaseDetailsBody = () => {
   return (
     <ContentLayout>
       <Flex gap={8} direction="column" alignItems="stretch">
-        <Flex justifyContent="space-between" alignItems="flex-start">
-          <Flex>
-            <SingleSelect
-              aria-label={formatMessage({
-                id: 'pages.ReleaseDetails.groupBy.label',
-                defaultMessage: 'Group by',
-              })}
-              customizeContent={(value) =>
-                formatMessage(
-                  {
-                    id: `pages.ReleaseDetails.groupBy.label}`,
-                    defaultMessage: `Group by {groupBy}`,
-                  },
-                  {
-                    groupBy: value,
-                  }
-                )
-              }
-              value={query?.groupBy || 'contentType'}
-              onChange={(value) => setQuery({ groupBy: value as ReleaseActionGroupBy })}
-            >
-              {GROUP_BY_OPTIONS.map((option) => (
-                <SingleSelectOption key={option.value} value={option.value}>
-                  {option.label}
-                </SingleSelectOption>
-              ))}
-            </SingleSelect>
-          </Flex>
+        <Flex>
```

Comment:
why is this flex necessary, since it only has 1 child?

Reply:
Because the flex parent has `alignItems="stretch"` for the table to occupied all the available space. If I don't add this flex the groupBy select also stretch on all the width

Reply:
would a simple div work?

Reply:
If you put a div with nothing it would inherit the styles from its parents and you would end with the same problem, so you would need to add a `display: flex` to that div and end with the same result than using `<Flex>`